### PR TITLE
July 2021 news

### PR DESCRIPTION
**Pull request by maelle**
Waiting for a blurb by @mpadge :wink:

### CODE REVIEW

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+<!-- blabla -->
+You can read this post [on our blog](/blog/2021/07/23/ropensci-news-digest-july-2021).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+<!-- to be curated manually -->
+
+Great news for fans of statistics and software!
+rOpenSci's peer-review system has now [officially expanded](https://ropensci.org/blog/2019/07/15/expanding-software-review/), and we are thrilled to be able to **review statistical software in seven categories**.
```
Comment (noamross) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:31]:
```suggestion
rOpenSci's peer-review system has now [expanded](https://ropensci.org/blog/2019/07/15/expanding-software-review/) to include review of packages implementing statistical routines and algorithms.  We are thrilled to start this new chapter, for which we've developed a series of new standards and tools, and recruited a new [board of experts](https://stats-devguide.ropensci.org) to oversee.
```

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+Authors of statistical software should begin by reading the [*Stats Software Dev Guide*](https://stats-devguide.ropensci.org), which describes the seven categories, along with the procedures for preparing statistical software for peer review.
```
Comment (noamross) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:32]:
```suggestion
Authors of statistical software should begin by reading the [*Stats Software Dev Guide*](https://stats-devguide.ropensci.org), which describes the categories of statistical software we now accept (Regression, Machine Learning, Exploratory Data Analysis, and more!), along with the procedures for preparing statistical software for peer review. It also introduces our automation tools, such as,  [the `pkgcheck` package](https://github.com/ropensci-review-tools/pkgcheck), which confirms that statistical software is ready to be submitted for review.
```

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+One of our new tools developed for this system,  [the `pkgcheck` package](https://github.com/ropensci-review-tools/pkgcheck), confirms that statistical software is ready to be submitted for review.
```
Comment (noamross) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:33]:
```suggestion
```

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+Submissions of statistical software begin with a pre-submission enquiry on our main [software review repository](https://github.com/ropensci/software-review).
```
Comment (noamross) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:34]:
```suggestion
Authors submitting of statistical software to review should start by opening _pre-submission inquiry_ on our main [software review repository](https://github.com/ropensci/software-review).  Inquiries are welcome well before packages are ready for submittal.  We hope authors make use of our standards and tools throughout the development process so as to make review straightforward.
```

**Review by noamross — CHANGES_REQUESTED** [review]
Very exciting! I've fleshed this out a bit.

**Comment by maelle** [comment]
Thank you @noamross! I've added your edits.

**Comment by mpadge** [comment]
Yes, indeed - thank you very much @noamross; now sounds much more exciting that my initial version!

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+If you follow the Tidyverse example, then it'd make sense to _test_ your package in those R versions on [continuous integration](https://devguide.ropensci.org/ci.html).
+
+Now if your choice of a recent R version as a minimal version stems from a newer base function, you might be interested in the [backports package](https://github.com/r-lib/backports) maintained by Michel Lang, that provides reimplementations of Functions Introduced Since R-3.0.0.
+Its README explains how to use it [in packages](https://github.com/r-lib/backports#in-packages).
+The README also has a list of the re-implementations, which is fun to look at for R history sake!
+
+### Roles in package development & DESCRIPTION
+
+It takes a village to... write an R package, or at least it can: creator, authors, reviewers, contributors, copyrightholders, etc.
+In an R package DESCRIPTION you can use the [roles](https://www.loc.gov/marc/relators/relacode.html) found in `utils:::MARC_relator_db_codes_used_with_R`: aut, com, cph, cre, ctb, ctr, dtc, fnd, rev, ths, trl.
```
Comment (mpadge) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:168]: At the end of that list of roles, maybe

>  The current list of roles accepted for R packages are always listed in the output of `?person`

**Review by mpadge — CHANGES_REQUESTED** [review]

**Review by stefaniebutland — COMMENTED** [review]
I like the call to action of digging in the archive!

I made minor grammar changes to para 1.

**Comment by maelle** [comment]
Thanks @stefaniebutland! :pray:

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+* [How to create your personal CRAN-like repository on R-universe](/blog/2021/06/22/setup-runiverse) by Jeroen Ooms. Explanation of the few simple steps needed to launch your universe (create a git repository with a special JSON file, install a GitHub app).
+
+* [How We Curate Our Monthly Newsletter](/blog/2021/06/24/news-meta) by Maëlle Salmon. Our workflow for preparing and sending our monthly newsletter, partly automatically.
+
+* [New package katex: rendering math to HTML and MathML in R](/blog/2021/07/13/katex-release) by Jeroen Ooms. Convert LaTeX math expressions to HTML and MathML for use in Markdown documents or package documentation.
+
+## Citations
+
+No new citations added to our database this month ([browse all citations](/citations)).
+Thanks for citing our tools!
```
Comment (mpadge) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:129]: @maelle Might be worth updating your code to remove "Thanks for citing our tools" when there are no citations (like this month)?

Comment (maelle) [reply content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:129]: ah, does it sound passive agressive?

Comment (mpadge) [reply content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:129]: Not passive aggressive exactly, just odd issuing a thanks for nothing.

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+Authors submitting statistical software to review should start by opening a _pre-submission inquiry_ on our main [software review repository](https://github.com/ropensci/software-review).
+Inquiries are welcome well before packages are ready for submittal.
+We hope authors make use of our standards and tools throughout the development process so as to make review straightforward.
+To volunteer as a _reviewer_, fill our [short form](/software-reviewer).
+For any question, you can post on the [rOpenSci forum](https://discuss.ropensci.org/c/statistical-software-peer-review/28) or contact the statistical software team (Mark Padgham, mark\<at\>ropensci.org, and Noam Ross, ross\<at\>ecohealthalliance.org).
+
+On another topic, if you find yourself with more time for learning this summer, why not dig into the [rOpenSci community calls archive](https://ropensci.org/commcalls/)?
+Every topic comes with a 1-hour video, speakers' slides, links to related resources, and a document with questions and answers from participants.
+Since March 2020, video recordings have closed captions.
```
Comment (steffilazerte) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:43]:
```suggestion
As of March 2020, video recordings have closed captions.
```

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+Its README explains how to use it [in packages](https://github.com/r-lib/backports#in-packages).
+The README also has a list of the re-implementations, which is fun to look at for R history sake!
+
+### Roles in package development & DESCRIPTION
+
+It takes a village to... write an R package, or at least it can: creator, authors, reviewers, contributors, copyrightholders, etc.
+In an R package DESCRIPTION you can use the [roles](https://www.loc.gov/marc/relators/relacode.html) found in `utils:::MARC_relator_db_codes_used_with_R`: aut, com, cph, cre, ctb, ctr, dtc, fnd, rev, ths, trl.
+The current list of roles accepted for R packages are always listed in the output of `?person`.
+The official recognition of the reviewer role is something rOpenSci [asked for](/blog/2018/03/16/thanking-reviewers-in-metadata/) and that we recommend in our [development guide](https://devguide.ropensci.org/building.html#authorship).
+We also advise to pair names with [ORCID IDs](/blog/2018/10/08/orcid/).
```
Comment (steffilazerte) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:171]:
```suggestion
We also advise pairing names with [ORCID IDs](/blog/2018/10/08/orcid/).
```

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+Inquiries are welcome well before packages are ready for submittal.
+We hope authors make use of our standards and tools throughout the development process so as to make review straightforward.
+To volunteer as a _reviewer_, fill our [short form](/software-reviewer).
+For any question, you can post on the [rOpenSci forum](https://discuss.ropensci.org/c/statistical-software-peer-review/28) or contact the statistical software team (Mark Padgham, mark\<at\>ropensci.org, and Noam Ross, ross\<at\>ecohealthalliance.org).
+
+On another topic, if you find yourself with more time for learning this summer, why not dig into the [rOpenSci community calls archive](https://ropensci.org/commcalls/)?
+Every topic comes with a 1-hour video, speakers' slides, links to related resources, and a document with questions and answers from participants.
+Since March 2020, video recordings have closed captions.
+
+Find out about [events](/events).
```
Comment (steffilazerte) [inline content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:45]: Probably not for this newsletter, but would it make sense to actually have an events section where upcoming events are listed?

**Review by steffilazerte — COMMENTED** [review]
A couple of super minor wording suggestions, looks good!

File: content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md (hunk 9 above)
Comment (maelle) [reply content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:45]: That's a good idea, we've only done that for users but I like that!

Comment (maelle) [reply content/blog/2021-07-23-ropensci-news-digest-july-2021/index.md:45]: https://github.com/ropensci/biweekly/issues/75

**Comment by maelle** [comment]
Thank you @steffilazerte!

**Pull request merged**